<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduledTask</class>
 <widget class="QWidget" name="ScheduledTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Applications list</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="check_action2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>- not foreground</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;a href=&quot;https://github.com/PerMalmberg/libcron#supported-formatting&quot;&gt;help&lt;/a&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="sub_action2"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QPlainTextEdit" name="appslist"/>
   </item>
   <item row="6" column="3">
    <widget class="QToolButton" name="reload_sub_actions2">
     <property name="text">
      <string>⟳</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QComboBox" name="action"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Action fo app is</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="sub_action"/>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLabel" name="statusapps">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Waiting on start and Aplication list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Cron expression</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="next_run">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="reload_sub_actions">
     <property name="toolTip">
      <string>Reload profile list</string>
     </property>
     <property name="text">
      <string>⟳</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="cron_line">
     <property name="toolTip">
      <string>Cron expression</string>
     </property>
     <property name="text">
      <string>* * * * * ?</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="check_action">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>- foreground</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QComboBox" name="cron_presets"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Next run in:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QComboBox" name="action2"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Action fo app is</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Preset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
